calendar-sheet-timesheet/README.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Manage your timesheet with the calender and sheets
```

Comment:
I like the title you have in your Medium article but let's go a little shorter:
"Record time and activities in Calendar and Sheets"
or maybe:
"Sync your time tracker to a Calendar and Sheet"

There's already a 'timesheets' solution so I'd like rephrase